### PR TITLE
Md sa2 22 22

### PR DESCRIPTION
**Pull request by Keldanovich**
Ended task

### CODE REVIEW

File: .github/workflows/action.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+name: Check PR repository
```
Comment (pluhin) [inline .github/workflows/action.yaml:1]: Please remove this from your PR, or add to your folder, or add to the report as code section
It should be outside your folder

**Review by pluhin — CHANGES_REQUESTED** [review]

File: .github/workflows/action.yaml (hunk 1 above)
Comment (pluhin) [reply .github/workflows/action.yaml:1]: @Keldanovich  could you please share updates?

Comment (Keldanovich) [reply .github/workflows/action.yaml]: done

**Comment by Keldanovich** [comment]
Added tasks

**Comment by Keldanovich** [comment]
Will recreate pr.

**Pull request closed without merging**